frame/staking/src/lib.rs

```diff
@@ -1576,19 +1660,30 @@ decl_module! {
 			}
 		}
 
-		/// The ideal number of validators.
-		#[weight = 5_000_000]
+		/// Sets the ideal number of validators.
+		///
+		/// The dispatch origin must be Root.
+		///
+		/// # <weight>
+		/// Base Weight: 1.717 µs
+		/// Write: Validator Count
+		/// # </weight>
+		#[weight = 2 * WEIGHT_PER_MICROS + T::DbWeight::get().writes(1)]
```

Comment:
should this not be operational (regardless of what it was before)

Reply:
see https://github.com/paritytech/substrate/issues/6040

I think some should be operational, but i dunno if I can make that decision. Better to bring it up with Gav or a public channel

Reply:
fair enough